pkg/capabilities/loopserver/loopserver.go

```diff
@@ -0,0 +1,31 @@
+// TODO this file is copy/pasted from capabilities repo so I can use it without a circular dependency in the generated code for testing...
+
+package loopserver
+
+import (
+	"github.com/hashicorp/go-plugin"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+)
+
+func Serve[T loop.StandardCapabilities](serviceName string, createPluginServer func(logger.Logger) T) {
```

Comment:
what is the loop server used for?

Reply:
This allows you to run a standard capability inside a standard loop. I borrowed a lot of it from the capabilities repo. It works with the generated code for the capability developer